packages/salesforcedx-vscode-apex/README.md

```diff
@@ -35,6 +37,13 @@ The Problems view in the bottom pane also lists the syntax errors. Double-click
 The Apex outline view shows the structure of the Apex class or trigger that’s open in the editor. For a list of the symbols in your file, press Cmd+Shift+O (macOS) or Ctrl+Shift+O (Windows or Linux). To jump to one of the symbols, select it in the list.  
 ![Outline view, showing the symbols in an Apex class](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-apex/images/apex_outline.png)
 
+## Enable Code Smartness for SObjects
```

Comment:
Custom fields are definitely worth mentioning in this section though. Also, I think we should mention the issue objects not being visible due to default field level security after a source:push.